websites/F/FR-Minecraft/dist/metadata.json

```diff
@@ -0,0 +1,32 @@
+{
+  "$schema": "https://schemas.premid.app/metadata/1.3",
+  "author": {
+    "name": "fantomitechno 🦊",
+    "id": "563749920683720709"
+  },
+  "url": "fr-minecraft.net",
+  "description": {
+    "en": "FR-Minecraft : News Minecraft, tips, tools et all the help necessary to play Minecraft !",
+    "fr": "FR-Minecraft : L'actualité Minecraft, des astuces, des outils et toute l'aide necessaire pour jouer à Minecraft !."
```

Comment:
Please remove ``FR-Minecraft :`` from description

Reply:
also remove the space before the ! at the end.

Reply:
Forget this I change it